### ISSUE TITLE
BUG: Install.cmd won't install on Windows

### ISSUE THREAD

**Issue by sachin-gupta** [tj/git-extras#731]
install.cmd uses `MORE +2` command which throws error `Too many arguments` as extra command switch `/E` is missing. This should be `MORE /E +2`, see help on more.exe. Tried on system and verified. Have attched corrected snippet:

**Snippet**

```
echo Installing binaries...
FOR /R "%GITEXTRAS%\bin" %%i in (*.*) DO (
    ECHO "WRITING = %%~ni"
    ECHO #^^!/usr/bin/env bash > "%PREFIX%\bin\%%~ni"
    TYPE "%GITEXTRAS%\helper\reset-env" >> "%PREFIX%\bin\%%~ni"
    TYPE "%GITEXTRAS%\helper\git-extra-utility" >> "%PREFIX%\bin\%%~ni"
    TYPE "%GITEXTRAS%\helper\is-git-repo" >> "%PREFIX%\bin\%%~ni"
    MORE /E +2 "%GITEXTRAS%\bin\%%~ni" >> "%PREFIX%\bin\%%~ni"
)

FOR %%i in (%COMMANDS_WITHOUT_REPO%) DO (
    ECHO "WRITING = %%i"
    ECHO #^^!/usr/bin/env bash > "%PREFIX%\bin\%%i"
    TYPE "%GITEXTRAS%\helper\reset-env" >> "%PREFIX%\bin\%%i"
    TYPE "%GITEXTRAS%\helper\git-extra-utility" >> "%PREFIX%\bin\%%i"
    MORE /E +2 "%GITEXTRAS%\bin\%%i" >> "%PREFIX%\bin\%%i"
)
```

**Help on More.exe**

```bash
C:> more /?
Displays output one screen at a time.

MORE [/E [/C] [/P] [/S] [/Tn] [+n]] < [drive:][path]filename
command-name | MORE [/E [/C] [/P] [/S] [/Tn] [+n]]
MORE /E [/C] [/P] [/S] [/Tn] [+n] [files]

    [drive:][path]filename  Specifies a file to display one
                            screen at a time.

    command-name            Specifies a command whose output
                            will be displayed.

    /E      Enable extended features
    /C      Clear screen before displaying page
    /P      Expand FormFeed characters
    /S      Squeeze multiple blank lines into a single line
    /Tn     Expand tabs to n spaces (default 8)

            Switches can be present in the MORE environment
            variable.

    +n      Start displaying the first file at line n

    files   List of files to be displayed. Files in the list
            are separated by blanks.

    If extended features are enabled, the following commands
    are accepted at the -- More -- prompt:

    P n     Display next n lines
    S n     Skip next n lines
    F       Display next file
    Q       Quit
    =       Show line number
    ?       Show help line
    <space> Display next page
    <ret>   Display next line
```

**Comment by spacewander** [comment]
@sachin-gupta 
The `help` information doesn't mention that `/E` is required for `+n` option.
At least `MORE +2` works well in Windows 10.0.15063 (I don't have other Windows to test for).
Maybe the problem only affects some versions of Windows. What is your version of Windows?

**Comment by sachin-gupta** [comment]
Updated a week before

⁣Sachin Gupta

**Comment by spacewander** [comment]
@sachin-gupta 
Do you mean the recent version of Windows 10?

**Comment by raeffs** [comment]
I can verify this on Windows 10.0.17134 Build 17134

**Comment by sachin-gupta** [comment]
Submitted [here](https://github.com/tj/git-extras/pull/732), A very nice product.

**Comment by spacewander** [comment]
Consider fixed as #732 is merged.
